frontend/src/components/Reports/Routine.js

```diff
@@ -82,8 +82,7 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link:
-            config.serverBaseUrl + "/Report?type=patient&report=referredOut",
+          link:"/RoutineReport?type=patient&report=referredOut",
```

Comment:
@konavivekramakrishna , 
@mherman22 is right . you worked on the wrong report menu item . 
YOu were supposed to work on [this ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/Reports/Routine.js#L133-L144)  ie it shoulb become 
`"/RoutineReport?type=patient&report=CISampleRoutineExport",`